backend/docs/db/schema.sql

```diff
@@ -464,6 +464,11 @@ CREATE VIEW public.sequence_entries_view AS
     sepd.finished_processing_at,
     sepd.processed_data,
     (sepd.processed_data || em.joint_metadata) AS joint_metadata,
+        CASE
+            WHEN se.is_revocation THEN ( SELECT current_processing_pipeline.version
```

Comment:
Why don't we just send the pipeline version here for everything? Maybe I'm missing something, might be good to explain why the complexity is necessary?

Reply:
discussed in https://github.com/loculus-project/loculus/pull/3433#discussion_r1882006866 (I haven't really followed the discussion)